Code/Tools/FBuild/FBuildCore/FBuildOptions.cpp

```diff
@@ -173,6 +173,26 @@ FBuildOptions::OptionsResult FBuildOptions::ProcessCommandLine( int argc, char *
                 m_Args += argv[ sizeIndex ];
                 continue;
             }
+            else if (thisArg == "-distcompressionlevel")
```

Comment:
I can fix this up post integration, so it's not a huge problem, but adherence to the coding style (specifically whitespace in braces in this instance) would make integrations a bit easier.